### PR TITLE
Balloonify and removes some `to_chat` spam for miners

### PR DESCRIPTION
**Pull request by GuillaumePrata**
## About The Pull Request
Playing as miner and paying attention to the radio is pure pain, so many of your job related actions create useless spam messages inside your chat/radio tab (unless you split them, which creates other issues) so this PR changes a few of them:

1 - PKAs won't send a useless * click *  message to your chat window when it's not ready to fire. It already has a different and clear sound that does it's job.
2 - Guns won't send a message to your chat with "you shoot the gun". You clicked your mouse, you hear the sound, you see the projectile leave and you already get a message logged there with what you hit.
3 - Digging with a pickaxe/shovel/hand is a balloon alert and the message for when you finish digging a wall is removed as you just seen the wall of basalt disappear so you know what happened...

## Why It's Good For The Game
As a miner most of your interaction with the station is over the radio, less useless spam on the chat tab will hopefully help make mining feel less like a single player game.

## Changelog

:cl: Guillaume Prata
qol: Miners rejoice! A few of the mostly useless and spammy messages that get send to your chat were changed to balloon alerts or straight up removed. Hopefully, you will have an easier time listening to the radio chatter now.
/:cl:

### CODE REVIEW

File: code/modules/projectiles/gun.dm [hunk 1]
```diff
@@ -173,7 +173,6 @@
 					PBT.throw_at(throw_target, pb_knockback, 2)
 			else if(!tk_firing(user))
 				user.visible_message(span_danger("[user] fires [src]!"), \
-								span_danger("You fire [src]!"), \
```
Comment (Mothblocks) [inline code/modules/projectiles/gun.dm:176]: I don't think this removes the message, I think this'll just make it so "You hear a gunshot!" is the new message they see

File: code/game/turfs/open/asteroid.dm [hunk 2]
```diff
@@ -71,12 +71,11 @@
 		if(!isturf(user.loc))
 			return
 
-		to_chat(user, span_notice("You start digging..."))
+		balloon_alert(user, "start digging...")
```
Comment (Mothblocks) [inline code/game/turfs/open/asteroid.dm:74]:
```suggestion
		balloon_alert(user, "digging...")
```

File: code/game/turfs/closed/minerals.dm [hunk 3]
```diff
@@ -142,10 +140,10 @@
 	..()
 
 /turf/closed/mineral/attack_alien(mob/living/carbon/alien/user, list/modifiers)
-	to_chat(user, span_notice("You start digging into the rock..."))
+	balloon_alert(user, "digging into the rock...")
 	playsound(src, 'sound/effects/break_stone.ogg', 50, TRUE)
 	if(do_after(user, 4 SECONDS, target = src))
-		to_chat(user, span_notice("You tunnel into the rock."))
+		balloon_alert(user, "finish digging")
```
Comment (Mothblocks) [inline code/game/turfs/closed/minerals.dm:146]:
```suggestion
		balloon_alert(user, "finished digging")
```

File: code/game/turfs/closed/minerals.dm [hunk 4]
```diff
@@ -101,12 +100,11 @@
 	TIMER_COOLDOWN_START(src, REF(user), hand_mine_speed)
 	var/skill_modifier = 1
 	skill_modifier = user?.mind.get_skill_modifier(/datum/skill/mining, SKILL_SPEED_MODIFIER)
-	to_chat(user, span_notice("You start pulling out pieces of [src]..."))
+	balloon_alert(user, "start pulling out pieces...")
```
Comment (Mothblocks) [inline code/game/turfs/closed/minerals.dm:103]:
```suggestion
	balloon_alert(user, "pulling out pieces...")
```

File: code/game/turfs/closed/minerals.dm [hunk 5]
```diff
@@ -80,13 +80,12 @@
 
 		TIMER_COOLDOWN_START(src, REF(user), tool_mine_speed)
 
-		to_chat(user, span_notice("You start picking..."))
+		balloon_alert(user, "start picking...")
```
Comment (Mothblocks) [inline code/game/turfs/closed/minerals.dm:83]:
```suggestion
		balloon_alert(user, "picking...")
```

**Review by Mothblocks — CHANGES_REQUESTED** [review]

**Comment by Striders13** [comment]
Hey can you make the "Your heart relaxes..." balloon alert from luxury mining pen into to_chat thing? It's super easy to miss and I keep blowing myself up thinking the pen's effect ran out.

File: code/modules/projectiles/gun.dm (hunk 1 above)
Comment (GuillaumePrata) [reply code/modules/projectiles/gun.dm:176]: It did but I manage to figure out the proper way to make these be quiet to the user.
Thanks.

**Comment by GuillaumePrata** [comment]
> Hey can you make the "Your heart relaxes..." balloon alert from luxury mining pen into to_chat thing? It's super easy to miss and I keep blowing myself up thinking the pen's effect ran out.

The proper way to inform something that vital should be a buff icon and not just a `balloon_alert` or `to_chat` message.

**Comment by Striders13** [comment]
> The proper way to inform something that vital should be a buff icon and not just a `balloon_alert` or `to_chat` message.

Would be cool.

File: code/modules/projectiles/gun.dm [hunk 6]
```diff
@@ -159,21 +159,29 @@
...
 		if(message)
 			if(tk_firing(user))
-				visible_message(span_danger("[src] fires itself[pointblank ? " point blank at [pbtarget]!" : "!"]"), \
-								blind_message = span_hear("You hear a gunshot!"), \
-								vision_distance = COMBAT_MESSAGE_RANGE)
+				visible_message(
+					span_danger("[src] fires itself[pointblank ? " point blank at [pbtarget]!" : "!"]"),
+					blind_message = span_hear("You hear a gunshot!"),
+					vision_distance = COMBAT_MESSAGE_RANGE
+					)
```
Comment (Mothblocks) [inline code/modules/projectiles/gun.dm:166]: Move all these back an indentation

File: code/modules/projectiles/gun.dm [hunk 7]
```diff
@@ -160,28 +160,28 @@
...
-					span_danger("[user] fires [src]!"),
-					blind_message = span_hear("You hear a gunshot!"),
-					vision_distance = COMBAT_MESSAGE_RANGE,
-					ignored_mobs = user
-					)
+				span_danger("[user] fires [src]!"),
+				blind_message = span_hear("You hear a gunshot!"),
+				vision_distance = COMBAT_MESSAGE_RANGE,
+				ignored_mobs = user
+				)
```
Comment (Mothblocks) [inline code/modules/projectiles/gun.dm:184]: No, not like this. As in, make it look like this:

```dm
call(
    a,
    b,
    c,
)
```

File: code/modules/projectiles/gun.dm [hunk 8]
```diff
@@ -159,22 +159,29 @@
...
 					PBT.throw_at(throw_target, pb_knockback, 2)
 			else if(!tk_firing(user))
-				user.visible_message(span_danger("[user] fires [src]!"), \
-								span_danger("You fire [src]!"), \
-								span_hear("You hear a gunshot!"), COMBAT_MESSAGE_RANGE)
+				user.visible_message(
+									span_danger("[user] fires [src]!"),
+									blind_message = span_hear("You hear a gunshot!"),
+									vision_distance = COMBAT_MESSAGE_RANGE,
+									ignored_mobs = user
```
Comment (Mothblocks) [inline code/modules/projectiles/gun.dm:183]: only indent once not 4 times x_x

File: code/game/turfs/closed/minerals.dm [hunk 9]
```diff
@@ -101,12 +100,11 @@
 	TIMER_COOLDOWN_START(src, REF(user), hand_mine_speed)
 	var/skill_modifier = 1
 	skill_modifier = user?.mind.get_skill_modifier(/datum/skill/mining, SKILL_SPEED_MODIFIER)
-	to_chat(user, span_notice("You start pulling out pieces of [src]..."))
+	balloon_alert(user, "pulling out pieces...")
```
Comment (Fikou) [inline code/game/turfs/closed/minerals.dm:103]: pieces of what?

Comment (GuillaumePrata) [reply code/game/turfs/closed/minerals.dm:103]: The snow/sand wall they just clicked.

**Pull request merged**